### PR TITLE
Ut for route column

### PR DESCRIPTION
**Pull request by garimaprachi**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: tests/phpunit/tests/test-razorpay-route.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+        $this->assertSame('1234567890', $this->instance->column_default($item,'contact'));
+
+        $this->assertSame('Processing', $this->instance->column_default($item,'status'));
+
+        $this->assertSame('0001142', $this->instance->column_default($item,'reversal_id'));
+
+        $this->assertSame('ABC', $this->instance->column_default($item,'la_name'));
+
+        $this->assertSame('0987654321', $this->instance->column_default($item,'la_number'));
```
Comment (sonyy-m) [inline tests/phpunit/tests/test-razorpay-route.php:175]: leave space after comma

File: tests/phpunit/tests/test-razorpay-route.php [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+        $result = $this->instance->column_default($item, 'Razorpay');
+        
+        $this->assertStringContainsString('[email] => abc.xyz@razorpay.com', $result);
+    }
+}
```
Comment (sonyy-m) [inline tests/phpunit/tests/test-razorpay-route.php:181]: Add EOL char

**Review by sonyy-m — CHANGES_REQUESTED** [review]

File: tests/phpunit/tests/test-razorpay-route.php (hunk 2 above)
Comment (garimaprachi) [reply tests/phpunit/tests/test-razorpay-route.php]: Done

**Review by sonyy-m — APPROVED** [review]
LGTM

**Pull request merged**